res/controllers/Pioneer-DDJ-400.midi.xml

```diff
@@ -0,0 +1,3291 @@
+<?xml version="1.0" encoding="utf-8"?>
+<MixxxMIDIPreset schemaVersion="1" mixxxVersion="2.2">
+    <info>
+        <name>Pioneer DDJ-400</name>
+        <author>Warker/nschloe/dj3730/jusko</author>
+        <description>Midi Mapping for the Pioneer DDJ-400</description>
+        <wiki>https://www.mixxx.org/wiki/doku.php/pioneer_ddj-400</wiki>
```

Comment:
After merging this in 2.3, we need a `<manual>` tag here.

Reply:
> Can you test if this mapping works on 2.3 without changes? 

It works fine as is. I'm going to give it all a thorough test on both versions today...

> I anything not mapped or not working as expected? In that case I suggest a "Known Issues" section at the bottom of the manual page.

Ok. Will see if anything needs to be mentioned as such.

Reply:
> After merging this in 2.3, we need a <manual> tag here.

Just to clarify @Holzhaus, must I:

1. Add `<manual>https://manual.mixxx.org/2.3/en/hardware/controllers/pioneer_ddj_400.html</manual>` to _this_ PR?

and

2. Remove or retain the `<wiki>` tag?

Reply:
No, don't add it to this PR (at least as long as the target branch is 2.2).
Instead, file a new PR against the 2.3 after this has been merged. 

Also, the XML needs to look like this IIRC:
```xml
<manual>pioneer_ddj_400</manual>
```

Reply:
Does the wiki page contain useful information not found in the manual? If not, remove it.